qa/rpc-tests/test_framework/blockchainhelper.py

```diff
@@ -27,6 +27,16 @@
 """ Number of bits used to represent the field modulus """
 MODULUS_BITS = 255
 
+"""
+Dictionary of default parameters that can be used when calling 'create_sidechain'.
+"""
+SidechainParameters = {
+    "DEFAULT_SC_V0":                { "version": 0 },
+    "DEFAULT_SC_V1":                { "version": 1 },
+    "DEFAULT_SC_V2_CEASABLE":       { "version": 2, "withdrawalEpochLength": 10 },
+    "DEFAULT_SC_V2_NON_CEASABLE":   { "version": 2, "withdrawalEpochLength": 0 }
```

Comment:
Could be worth to add `"wCeasedVk": ""` for non-ceasable sidechains?

Reply:
I like this suggestion as it also avoids having an ugly check inside `from_rpc_args()` to eventually unset `wCeasedVk`.
Fixed in the latest commit, thanks!